[DOCS] Remove all mentions of profilers in public docs (#8825)
great-expectations/great_expectations: docs/docusaurus/docs/terms/batch.md

A Batch is a selection of records from a <TechnicalTag relative="../" tag="data_asset" text="Data Asset" />.

> A Batch provides a consistent interface for describing specific data from any <TechnicalTag relative="../" tag="datasource" text="Data Source" />, to support building <TechnicalTag relative="../" tag="metric" text="Metrics" />, and <TechnicalTag relative="../" tag="validation" text="Validation" />.

Reviewer: ```suggestion
A Batch provides an interface for describing specific data from any <TechnicalTag relative="../" tag="datasource" text="Data Source" />, and supports the creation of <TechnicalTag relative="../" tag="metric" text="Metrics" />, and <TechnicalTag relative="../" tag="validation" text="Validations" />.
```

## Access

> You can access a Batch through the Data Asset `get_batch_list_from_batch_request` method. You typically will not need to access the Batch directly. Instead, you will pass a Batch Request to a Expectations object such as a Validator or Checkpoint, which will then do something in response to the Batch's data.

Reviewer: ```suggestion
You can use the `get_batch_list_from_batch_request` method to access a Batch through the Data Asset. You don't typically access the Batch directly. Instead, you pass a Batch Request to a Validator or a Checkpoint.
```
